src/common/validators.ts

```diff
@@ -11,3 +11,14 @@ export function validateInputLength(text: string, maxLength: number): string {
 
   return text
 }
+
+export function split(text: string, delimiter: string): string[] {
```

Comment:
Could be a one-liner? 🙂 

`text.split(delimiter).filter(item => item.length > 0)`

Reply:
yes, replaced